src/Strategy/Python/SetupPy.hs

```diff
@@ -45,8 +46,11 @@ type Parser = Parsec Void Text
 installRequiresParser :: Parser [Req]
 installRequiresParser = prefix *> entries <* end
   where
-  prefix  = skipManyTill anySingle (string "install_requires=[")
-  entries = between quote quote requirementParser `sepBy` char ','
-  end     = char ']'
+  prefix  = skipManyTill anySingle (symbol "install_requires") *> symbol "=" *> symbol "["
```

Comment:
These are the `setup.py` parser fixes (I didn't know about `symbol`/lexemes when I first wrote this)

Reply:
Do you have an example of what this fixes?

Reply:
- whitespace between things, e.g., `install_requires  =     \n[ 'foo']`
- trailing commas `install_requires=['foo',]`
- double-quotes used instead of single-quotes `install_requires=["foo"]`